NonEmergencyAmbulanceTransportation/R4/files/NonEmergencyAmbulanceTransportationRule-0.1.0.cql

```diff
@@ -27,8 +27,11 @@ define RESULT_InfoLink:
 define RESULT_QuestionnaireOrderUri:
   'Questionnaire/NonEmergencyAmbulanceTransportation'
 
-  define RESULT_QuestionnairePARequestUri:
-    'Questionnaire/NonEmergencyAmbulanceTransportationPARequest'
+define RESULT_QuestionnairePARequestUri:
```

Comment:
What is this for?

Reply:
Idk. It was already in there when I branched from master.